### ISSUE TITLE
[IgxBanner]: set banner role to status

### ISSUE THREAD

**Issue by RivaIvanova** [IgniteUI/igniteui-angular#14230]
## Description  

Should be aligned with the web components `igc-banner` https://github.com/IgniteUI/igniteui-webcomponents/pull/1185.
